rfcs/text/0000-data-processing-robustness.md

```diff
@@ -0,0 +1,75 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC Name
+
+DCP data processing robustness principles
+
+## Summary
+
+This document describes requirements for individual DCP components related to robust against incorrect, corrupt, or unexpected data.  It also lays out expectations for data integrity and error handling for DCP software.  It does not specify implementation details or technology.
+
+## Author(s)
+
+[Mark Diekhans](mailto:markd@ucsc.edu),
+[Tony Burdett](mailto:tburdett@ebi.ac.uk)
+
+## Shepherd
```

Comment:
If you do not require a shepherd to be assigned, then you could either delete this section or duplicate one of more of the authors here.